tests/Kernel/fixtures/rendering.yml

```diff
@@ -560,7 +560,8 @@
     equals:
       'div.ecl-file__detail-info div.ecl-file__title': "Example document"
       'div.ecl-file__detail-info div.ecl-file__description': 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer quis lorem tellus. Nullam sollicitudin suscipit diam, ac blandit ipsum tempor consectetur. Duis vitae pulvinar turpis. Donec maximus pharetra ex a ultricies.'
-      'div.ecl-file__detail-info div.ecl-file__detail-meta': 'RESOURCE TYPE | 2020-31-08'
+      'div.ecl-file__detail-info div.ecl-file__detail-meta span.ecl-file__detail-meta-item:nth-child(1)': 'RESOURCE TYPE'
```

Comment:
We could keep these 2 assertions plus the removed one since the component renders them as such.

Reply:
If we keep the removed one then the text will include html tags (the new span.ecl-file__detail-meta-item elements), that is why I removed it.